# Definition of continuous symmetry.
# Consists of rotation axis and offset.
# Symilarly to HappyPose object ContinuousSymmetry

geometry_msgs/Vector3 offset
geometry_msgs/Vector3 axis
